config/targets-metadata.yml

```diff
@@ -52,6 +19,11 @@ ctfe.pub:
     uri: https://ctfe.sigstore.dev
```

Comment:
Can we change the URIs to be `https://ctfe.sigstore.dev/test` and `https://ctfe.sigstore.dev/2022`?

Reply:
All of these are single string key-val pairs. I can dupe the target if need be, are both required?

Reply:
done!

Reply:
note for others: no one relies on the URI fields of custom metadata for discovery.